bin/post_compile

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+indent() {
+  sed "s/^/       /"
+}
+
+puts-step() {
+  echo "-----> $@"
+}
+
+puts-step "Running migrations..."
+
+python manage.py db upgrade 2>&1 | indent
```

Comment:
went with `set -o pipefail` since i've seen that used more often ([example](https://github.com/heroku/heroku-buildpack-python/blob/master/bin/compile#L8))